selenium/video/Dockerfile

```diff
@@ -1,21 +1,7 @@
-FROM alpine:3.12
+FROM alpine:3.18
 
-RUN apk add -U alpine-sdk && adduser -g "" -D -G abuild selenoid
-USER selenoid
-RUN git clone git://git.alpinelinux.org/aports /home/selenoid/aports
-ADD APKBUILD /home/selenoid/aports/community/ffmpeg/
-RUN cd /home/selenoid/aports/community/ffmpeg/ && \
-    abuild checksum && \
-    abuild unpack && \
-    abuild deps && \
-    abuild build
-USER root
-
-FROM alpine:3.12
-
-RUN apk add -U x264-libs sdl2 libxcb libbz2 xset pulseaudio-utils && \
+RUN apk add -U x264-libs sdl2 libxcb libbz2 xset pulseaudio-utils ffmpeg && \
```

Comment:
We know that `ffmpeg` is available as Alpine package. Could please show the size of resulting video recorder image and compare it with existing video recorder image. Our initial idea was to make it smaller.

Reply:
Yes true the size increase 132Mb from 37MB, but probably a ffmpeg  alpin dist package more tested, more reliable.
It is trade off.
* docker image size  
* apline pkg
   - more tested distribution ffmpeg pkg
   - no compile time => faster image build you prefer smaller image. I can move it separate PR or Cancel.
   - no risk that ports not compile

I feel there is always a higher risk to compile own binary from source using ports, than using just a dist binary. Where dist owner make sure it works well the pkg and probably run some tests and have graduated rollout of new versions, , but these are just my 2cents.

The +size 95mb that is bigger but it is still not huge.
But I can go along if you still prefer building, but I got actually an error during the build of ffmpeg with newer alpine version.